src/models/attemptModel.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+const attemptSchema = new mongoose.Schema({
+  test: {
```

Comment:
I'm inclined to rename `test` to `quiz` to keep the language consistent. What do you think?

Reply:
Renamed test to quiz to keep the language consistent.